test/mocks.ts

```diff
@@ -26,4 +26,35 @@ module Mocks {
       return true;
     }
   }
+
+  export class Scale<D, R> extends Plottable.Scale<D, R> {
```

Comment:
This gives more confidence that annotations will work on any type of scale; not just `Time` or `Linear` since all that is required from the scale is knowledge of the domain, range, and a scale mapping.